iceoryx_utils/test/moduletests/test_cxx_expected.cpp

```diff
@@ -82,6 +82,15 @@ TEST_F(expected_test, CreateError)
     EXPECT_THAT(sut.get_error().m_b, Eq(212));
 }
 
+TEST_F(expected_test, CreateValueAndMove)
+{
+    auto sut = expected<int, Test>::create_value(73);
+    auto movedValue = std::move(sut);
+    EXPECT_FALSE(sut.is_initialized());
+    EXPECT_TRUE(movedValue.is_initialized());
```

Comment:
Needs to be an assert otherwise we risk an invalid access.

Reply:
Fixed